telebot/async_telebot.py

```diff
@@ -424,7 +437,8 @@ async def _process_polling(self, non_stop: bool=False, interval: int=0, timeout:
             # show warning
             logger.warning("Setting non_stop to False will stop polling on API and system exceptions.")
 
-        self._user = await self.get_me()
+        if not self._user:
+            self._user = await self.get_me()
```

Comment:
We discussed that get_me should be optional.

Reply:
I will return this to the original version, where it is called by default?